docs/reference/errors-and-warnings/NU1105.md

```diff
@@ -12,10 +12,29 @@ f1_keywords:
 
 # NuGet Error NU1105
 
-<pre>Unable to read project information for 'ProjectFile'. The project file may be invalid or missing targets required for restore.</pre>
+<pre>Unable to read project information for 'ProjectFile'. The project file may be invalid/missing targets required for restore or the project might be unloaded.</pre>
 
 ### Issue
 The project file exists but no restore information was provided for it.
 
 ### Solution
-In Visual Studio, the error could mean that the project is unloaded, in which case reload the project. From the command line this could mean that the file is corrupt or that it doesn't contain the custom "after imports" target needed for restore to read the project. Check that the project file is valid and contains an "after imports" target.
\ No newline at end of file
+In Visual Studio, the error could mean that the project is unloaded, and thus NuGet was unable to load project information required to restore it. 
+
+#### Solution in VS 2019 16.0 or later:
+- Using a recent CLI version, restore the solution that includes that project from a command line prompt.
+
+or
+
+- Ensure the project is loaded as part of the current solution in Visual Studio, and then perform a solution restore inside of Visual Studio.
+
+**Details:**
+Unloaded projects, or projects that aren't included in the current VS Solution Filter, can still be part of a solution restore if they were restored from command line ([nuget.exe 5.0](https://docs.microsoft.com/en-us/nuget/release-notes/nuget-5.0-rtm) or later) or Visual Studio 16.0 or later. In those cases, NuGet writes obj\<projectFileName>.nuget.dgspec.json with all project information that NuGet will need later, in case the "live project information" isn't available.

```

Comment:
Consider changing the first sentence to make it clearer that the unloaded project is needed because of a project reference, and NuGet includes transitive information. Or maybe this information should be in the "Issue" section above.

unrelated to the PR, this makes me wonder how building the loaded project can work if binaries for the project reference haven't been built? Won't the user just get compile errors when trying to use types from the unloaded, uncompiled project?

Reply:
If you go back to the issue that added this feature, build already uses the last available built assemblies. 
And if they are out of date/not there, then it will fail regardless. 

Trick is, the build tried to get the last built assemblies. Restore never did that.